abs.yaml

```diff
@@ -0,0 +1,5 @@
+aggregate_check: false
+channels:
+  - sfe1ed40
```

Comment:
same thing as for libdate 3.0.1 PR

Reply:
@katietz thanks for the comment. Note that I do not intend to keep the abs.yaml file in the final build. I am probably doing the initial build wrong here, but I tend to develop in the services channel initially to ensure the build is correct. Once that passes, I remove abs.yaml, add it to the aggregate, and so forth. I apologize if I'm not doing things in the right order.

Reply:
removed abs.yaml, added to aggregate